first change to pig latin

noway19 opened the pull request:
added try again at end
@spacejam

noway19 committed:
first change to pig latin

added try again at end

spacejam commented on changestopiglatin.txt:
```diff
@@ -0,0 +1,13 @@
+if len(original) > 0 and original.isalpha():
+    word = original.lower()
+    first = word[0]
+    new_word = word+first+pyg
```
pep8 https://www.python.org/dev/peps/pep-0008/

spacejam commented on changestopiglatin.txt:
```diff
@@ -0,0 +1,13 @@
+    word = original.lower()
+    first = word[0]
+    new_word = word+first+pyg
+    new_word = new_word[1:len(new_word)]
```
While possible to bind `new_word` to the last version of `new_word` with a mutation, this is highly discouraged.  The more different things you call the same name, the more confusing your code gets.  Try to always avoid reusing the same variable name for different things.  If the same variable will be passed to different functions, then you SHOULD call it the same thing, but never call different things the same variable name.

spacejam replied:
You can skip the `len(new_word)` when you want to show the entire rest by just typing `new_word[1:]`

spacejam commented on changestopiglatin.txt:
```diff
@@ -0,0 +1,13 @@
+    new_word = new_word[1:len(new_word)]
+    print new_word
+else:
+    print 'empty try again'
```
this prints `empty try again` even when input was given, but it was not alphanumeric.

spacejam commented on changestopiglatin.txt:
```diff
@@ -0,0 +1,13 @@
+    first = word[0]
+    new_word = word+first+pyg
+    new_word = new_word[1:len(new_word)]
+    print new_word
```
I would have written this entire big block as:

```
if len(original) > 0 and original.isalpha():
    print (original[1:] + original[0] + 'ay').lower()
else:
    ...
```

Python is nice because it lets you write complicated things really concisely.

noway19 committed:
Revert "first change to pig latin"

This reverts commit e6c80bf25859617c5056d254cd564f965ed632ef.

noway19 committed:
Revert "Revert "first change to pig latin""

This reverts commit a82f4ebaab796fc46cd63a14dfc66268ebd8ac7e.

noway19 committed:
first feedback changes

changed new word to final word and simpplified len line, not sure if it
will work as typed

spacejam commented on firstfeedbacktopl.txt:
```diff
@@ -0,0 +1,13 @@
+    final_word = new_word[1:]
+    print final_word
+else:
+    print 'empty'
```
nuke this entire file

noway19 committed:
Delete firstfeedbacktopl.txt

noway19 committed:
Delete changestopiglatin.txt

spacejam commented:
You deleted all of the files!

noway19 commented:
You said delete all of this
On Nov 4, 2015 11:49 AM, "Tyler Neely" notifications@github.com wrote:

> You deleted all of the files!
> 
> —
> Reply to this email directly or view it on GitHub
> https://github.com/noway19/Pig-Latin-Translator/pull/1#issuecomment-153827871
> .

spacejam commented:
Miscommunication: I meant that when you make a PR, you should not copy entire files, and that the copy should not be here.  Instead, it's better to just modify existing files.  I'll show you what I mean later today on a google hangout or something if you have time, since this can be really weird until you see someone else do it.

noway19 commented:
Okay
On Nov 4, 2015 1:53 PM, "Tyler Neely" notifications@github.com wrote:

> Miscommunication: I meant that when you make a PR, you should not copy
> entire files, and that the copy should not be here. Instead, it's better to
> just modify existing files. I'll show you what I mean later today on a
> google hangout or something if you have time, since this can be really
> weird until you see someone else do it.
> 
> —
> Reply to this email directly or view it on GitHub
> https://github.com/noway19/Pig-Latin-Translator/pull/1#issuecomment-153861904
> .
